### PR TITLE
Reproduce and fix bug in Clojure renderer

### LINKED ISSUE #180: Clojure renderer is hanging or failing with large and large-ish diagram YAML files
In some cases, it hangs indefinitely. In some others, it returns correctly and _seems_ to have succeeded, but the PNG file that it returns is a visualization of the nearly-empty default diagram that is the default zero state of Structurizr Express (inline below).

Blocks #161

----

Here’s that zero-state PNG file:

structurizr-express

### PR DESCRIPTION
**Pull request by aviflax**
Wherein it would hang when rendering any diagram wherein the output PNG
was more than 1MB in data-size.

Fixes #180

### CODE REVIEW

**Comment by j-po** [comment]
Looks good to me. It looks like the fix was to set `:max-msg-size-mb` to a larger value, then you also added some logging changes to help you find the issue and a timeout so that similar errors don't hang the app in the future. Is that about right?

**Comment by aviflax** [comment]
@j-po yes, exactly right!

File: tool/src/main/fc4/util.clj [hunk 1]
```diff
@@ -112,3 +113,16 @@
   [message]
   {::anom/category ::anom/fault
    ::anom/message message})
+
+(defmacro with-timeout
+  "If the timeout elapses before the body has completed, a TimeoutException will be thrown with a
+  not-particularly-helpful message."
+  ;; To consider: perhaps we should include part of the body in the exception message?
+  {:derived-from "https://stackoverflow.com/q/6694530/7012"}
```
Comment (j-po) [inline tool/src/main/fc4/util.clj]: Is there tooling that uses this metadata?

Comment (aviflax) [reply tool/src/main/fc4/util.clj]: Nope, I made it up, ad-hoc.

**Pull request merged**